Author: Andrew Suffield <asuffield@debian.org>
Description: Add /etc/default/cdrdao to the set of config files read.

Index: cdrdao-1.2.3/dao/main.cc
===================================================================
--- cdrdao-1.2.3.orig/dao/main.cc	2009-05-04 17:49:52.000000000 +0200
+++ cdrdao-1.2.3/dao/main.cc	2010-07-04 14:11:32.000000000 +0200
@@ -2338,6 +2338,10 @@ int main(int argc, char **argv)
     if (settings->read(settingsPath) == 0)
 	log_message(3, "Read settings from \"%s\".", settingsPath);
 
+    settingsPath = "/etc/default/cdrdao";
+    if (settings->read(settingsPath) == 0)
+	log_message(3, "Read settings from \"%s\".", settingsPath);
+
     settingsPath = NULL;
 
     if ((homeDir = getenv("HOME")) != NULL) {
